UPGRADE.md

```diff
@@ -0,0 +1,81 @@
+# Upgrade Guide
+
+_All `MAJOR` version bumps will have upgrade notes posted here._
+
+[2022-04-27] 0.2.x to 1.x.xrc
+-----------------------------
+### CHANGED - Renammed ApiV2010 to Api.
+ApiV2010 has now been renamed to Api. This has caused a breaking change for all endpoints located under `rest/api/2010`.
+
+#### Send a SMS
+```go
+//0.2.x 
+client := twilio.NewRestClient()
+
+params := &api.CreateMessageParams{}
+params.SetFrom("+15017122661")
+params.SetBody("body")
+params.SetTo("+15558675310")
+
+resp, err := client.ApiV2010.CreateMessage(params)
+```
+```go
+//1.x.xrc
```

Comment:
Think we should write this as 1.0.0-rc.1 since this will get released in our 1st RC build.

Reply:
I can update it to `1.x.x-rc.x` but I think by changing this to `1.0.0-rc.1` some users could assume it's only a change that was applied to that specific version. We also use the same naming conventions in our [other upgrade guides](https://github.com/twilio/twilio-java/blob/main/UPGRADE.md#2020-09-28-7xx-to-8xx) to help make it clear that this  will apply for all versions higher than 0.2.x.

Reply:
Okay that makes sense, and we should keep it consistent with other guides. So feel free to disregard the suggestion.